lib/api/definitions/__tests__/CreatedMessage.test.ts

```diff
@@ -0,0 +1,349 @@
+import {
+  CreatedMessage,
+  isCreatedMessage,
+  toCreatedMessage
+} from "../CreatedMessage";
+
+import { toMessageBodyMarkdown } from "../MessageBodyMarkdown";
+
+import { MessageContent } from "../MessageContent";
+
+import { toMessageSubject } from "../MessageSubject";
+
+import { toFiscalCode } from "../FiscalCode";
+
+import { toTimeToLive } from "../TimeToLive";
+
+describe("CreatedMessage#toCreatedMessage", () => {
+  test("should returns a defined option for valid payloads", () => {
+    const s = toMessageSubject("Lorem ipsum dolor sit amet");
+    const m = toMessageBodyMarkdown(
+      // String long 90 characters.
+      "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt"
+    );
+
+    const messageContent: MessageContent = {
+      markdown: m.get,
+      subject: s.get
+    };
+
+    const message: CreatedMessage = {
+      content: messageContent,
+      fiscal_code: toFiscalCode("AAABBB01C01A000A").get,
+      id: "12345",
+      sender_organization_id: "Sender Organization",
+      time_to_live: toTimeToLive(3600).get
+    };
+
+    expect(toCreatedMessage(message).get).toEqual(message);
```

Comment:
what's the point here? `message` the compiler already checks that `message` is a `CreatedMessage` - I think you should change your testing strategy here, what you want to do is to generate a number of fixtures for valid and invalid objects (either manually like [we do here](https://github.com/teamdigitale/digital-citizenship-functions/blob/master/lib/models/__tests__/notification_event.test.ts#L43) or automatically as per @gunzip suggestion)